objects/src/accounts/storage/map.rs

```diff
@@ -1,14 +1,27 @@
+use vm_core::EMPTY_WORD;
+
 use super::{
-    AccountError, ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable, Word,
+    AccountError, ByteReader, ByteWriter, Deserializable, DeserializationError, Felt, Serializable,
+    Word,
 };
-use crate::crypto::{
-    hash::rpo::RpoDigest,
-    merkle::{InnerNodeInfo, LeafIndex, Smt, SmtLeaf, SmtProof, SMT_DEPTH},
+use crate::{
+    accounts::StorageMapDelta,
+    crypto::{
+        hash::rpo::RpoDigest,
+        merkle::{InnerNodeInfo, LeafIndex, Smt, SmtLeaf, SmtProof, SMT_DEPTH},
+    },
 };
 
 // ACCOUNT STORAGE MAP
 // ================================================================================================
 
+pub const EMPTY_STORAGE_MAP: Word = [
```

Comment:
Let's add a doc comment here. Also, should this be named `EMPTY_STORAGE_MAP_ROOT`?